database/factories/GmbHourFactory.php

```diff
@@ -15,20 +15,20 @@ class GmbHourFactory extends Factory
     public function definition()
     {
         return [
-            'monday_open'       => $this->faker->text,
-            'monday_close'      => $this->faker->text,
-            'tuesday_open'      => $this->faker->text,
-            'tuesday_close'     => $this->faker->text,
-            'wednesday_open'    => $this->faker->text,
-            'wednesday_close'   => $this->faker->text,
-            'thursday_open'     => $this->faker->text,
-            'thursday_close'    => $this->faker->text,
-            'friday_open'       => $this->faker->text,
-            'friday_close'      => $this->faker->text,
-            'saturday_open'     => $this->faker->text,
-            'saturday_close'    => $this->faker->text,
-            'sunday_open'       => $this->faker->text,
-            'sunday_close'      => $this->faker->text,
+            'monday_open'       => $this->faker->time('H:i A'),
+            'monday_close'      => $this->faker->time('H:i A'),
+            'tuesday_open'      => $this->faker->time('H:i A'),
+            'tuesday_close'     => $this->faker->time('H:i A'),
+            'wednesday_open'    => $this->faker->time('H:i A'),
+            'wednesday_close'   => $this->faker->time('H:i A'),
+            'thursday_open'     => $this->faker->time('H:i A'),
+            'thursday_close'    => $this->faker->time('H:i A'),
+            'friday_open'       => $this->faker->time('H:i A'),
+            'friday_close'      => $this->faker->time('H:i A'),
+            'saturday_open'     => $this->faker->time('H:i A'),
+            'saturday_close'    => $this->faker->time('H:i A'),
+            'sunday_open'       => $this->faker->time('H:i A'),
+            'sunday_close'      => $this->faker->time('H:i A'),
```

Comment:
Beautiful!

We should also create factory states for NULL values on a few days:

https://laravel.com/docs/8.x/database-testing#factory-states

Reply:
https://github.com/tipoff/locations/pull/110/commits/b6fac7785a52667608fb4106c491fa0e571d9cee

Created a `closed()` state in which you can pass a day that you want your factory to indicate as a closed today. After running the following via Tinker:

```
Tipoff\Locations\Models\GmbHour::factory()->closed('sunday')->create();
```

I had the following location generated via the factory with the GmbHours attached, which Sunday indicated as closed:

<img width="1141" alt="Screen Shot 2021-04-13 at 6 25 22 PM" src="https://user-images.githubusercontent.com/24723492/114633133-9cca0200-9c85-11eb-8b85-82172b3e2370.png">

Reply:
That really is slick. I want to use more factory states in other tests and examples for seeder data.